transformations/negate_strengthen/transformation.py

```diff
@@ -0,0 +1,443 @@
+import random
+from initialize import spacy_nlp
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import spacy
+from spacy.tokenizer import Tokenizer
+import pandas as pd
+import nltk
+from nltk.corpus import wordnet
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.metrics import edit_distance
+from nltk import pos_tag
+from pattern.en import conjugate as conjugate_en
+from typing import Union
+
+# defaults
+_RE_COMBINE_WHITESPACE = re.compile(r"\s+")
+MODAL_TYPE_LEMMA = ['be', 'to', 'have', 'get', 'do']
+MODAL_STRENGTHEN_DICT = {
+    'could': 'would',
+    'should': 'would',
+    'would': 'would',
+    'can': 'will',
+    'may': 'will',
+    'might': 'will',
+    'will': 'will'
+}
+
+# intialize
+try:
+    conjugate_en(verb='testing',tense='present',number='singular')
+except:
+    pass
+
+
+class NegateStrengthen(SentenceAndTargetOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.SENTIMENT_ANALYSIS]
+    languages = ["en"]
+    tgt_languages = ["en"]
+
+    def __init__(self, max_outputs=1, seed=0, verbose=False):
+        super().__init__(seed)
+
+        # downloads
+        nltk.download('wordnet')
+        nltk.download('averaged_perceptron_tagger')
+
+        # initialise
+        self.verbose = verbose
+        self.nlp = spacy.load("en_core_web_sm")
+        self.nlp.tokenizer = Tokenizer(
+            self.nlp.vocab, 
+            prefix_search=re.compile('''^\\$[a-zA-Z0-9]''').search
+            )
+        self.wnl = WordNetLemmatizer()
+        self.max_outputs = max_outputs
```

Comment:
Duplicate code: Parent class already have 'max_outputs' and 'seed' as a class variable, can we make use of it using super call instead of creating two more class variables?

Reply:
Ok, thanks for highlighting, just amended